### ISSUE TITLE
Save the results. 

### ISSUE THREAD

**Issue by iazdan** [DevashishPrasad/CascadeTabNet#125]
How to save the result in demo.py? 
"result = inference_detector(model, img)"

**Comment by mohit-217** [comment]
@iazdan do you need tabular coordinates for cropping the tables

**Comment by vaibhavkansallumiq** [comment]
yes i need the table coordinate as well as the cell coordinate for border tables
